data-prepper-plugins/opensearch-source/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+    id 'java'
+}
+
+group 'org.opensearch.dataprepper'
+version '2.3.0-SNAPSHOT'
```

Comment:
These can be removed

Reply:
Thanks @graytaylor0  for the review. The changes are incorporated.